es/criteria/document-maturity.md

```diff
@@ -22,20 +22,20 @@ Señalar claramente la madurez de una codebase ayuda a otros a decidir si reutil
 
 * Garantizar que otros usarán el código.
 
-## Cómo testear
+## Cómo probar
 
 * La codebase tiene una estrategia de versionado que está documentada.
 * Está claro dónde conseguir la versión más reciente.
 * La codebase no depende de ninguna codebase marcada con un estado menos maduro.
 
 ## Responsables de políticas y legislaciones: qué necesitan hacer
 
-* Al desarrollar políticas, hay que entender que cualquier código desarrollado debe ser probado y mejorado antes de ponerlo en servicio.
-* Considera la posibilidad de versionar los cambios de las políticas, especialmente cuando desencadenen nuevas versiones del código fuente.
+* Al desarrollar políticas, entender que cualquier código desarrollado debe ser probado y mejorado antes de ponerlo en servicio.
+* Considerar la posibilidad de versionar los cambios de las políticas, especialmente cuando desencadenen nuevas versiones del código fuente.
 
 ## Profesionales de la dirección de equipos: qué necesitan hacer
 
-* Asegúrate de que los servicios solo dependen de codebases de igual o mayor madurez que el servicio. Por ejemplo, no utilices una codebase beta en un servicio de producción ni una codebase prototype en un servicio beta.
+* Asegurarse de que los servicios sólo dependen de codebases de igual o mayor madurez que el servicio. Por ejemplo, no utilices una codebase beta en un servicio de producción ni una codebase prototype en un servicio beta.
```

Comment:
- Infitive, good one!
- "Solo" vs "sólo": I was myself a huge fan of the "sólo" with accent but unfortunately the Royal Academy of the Spanish Language has decided to keep just the one without an accent: https://dle.rae.es/solo?m=form